src/variants/lambdatreasury/storage.mligo

```diff
@@ -0,0 +1,185 @@
+// SPDX-FileCopyrightText: 2022 Tezos Commons
+// SPDX-License-Identifier: LicenseRef-MIT-TC
+
+#if !VARIANT_STORAGE
+#define VARIANT_STORAGE
+
+#include "variants/lambda/types.mligo"
+#include "error_codes.mligo"
+
+// Xtz transfer amount cannot be 0
+let zero_mutez_err_msg = "ZERO_MUTEZ"
+
+// Xtz transfer amount cannot be smaller than `min_xtz_amount`
+let too_small_xtz_err_msg = "LOW_XTZ"
+
+// Xtz transfer amount cannot be bigger than `max_xtz_amount`
+let too_large_xtz_err_msg = "HIGH_XTZ"
+
+let fail_proposal_check_ (msg : string) : unit =
+  ([%Michelson ({| { FAILWITH } |} : (nat * string) -> unit)]
+    (fail_proposal_check, msg) : unit)
```

Comment:
This is also duplicated in `lambdaregistry`, can it not be extracted in a common module? :thinking:

Reply:
Importing the existing common module seems to create circular imports, since the `types.mligo` module imports this module.